.debug/docker-compose-debug.yml

```diff
@@ -1,24 +1,27 @@
 version: "3"
 services:
-  postgres:
-    command: postgres -c shared_preload_libraries=pg_stat_statements -c pg_stat_statements.track=all
+  cockroachdb:
+    # Only use cockroachdb single-node clusters for non-production environment
+    image: cockroachdb/cockroach:latest-v23.1
+    command: start-single-node --insecure --store=attrs=ssd,path=/var/lib/cockroach/,size=20%
```

Comment:
is there a reason why we use `--insecure`?

Reply:
If we use secure cluster mode, the database will require a set of keys/certificates that should also be provided to the client. The client database URL should appear like this:
```
user:pass@10.xxx.xx.xx:26257/db_name?sslmode=verify-full&sslrootcert=/path/to/ca.crt"
```
This is a mandatory setup for the production server, but I think it might be too complex for the development mode. Wdyt?

Reply:
Ohh, I checked their docs again. Besides the `--insecure` option, there's a new preview feature `--accept-sql-without-tls` that lets clients choose not to implement TLS server authentication while maintaining encryption for the database traffic.
https://www.cockroachlabs.com/docs/v23.1/security-reference/transport-layer-security#accept-sql-without-tls-mode